.github/workflows/publish.yaml

```diff
@@ -49,3 +49,13 @@ jobs:
           tags: |
             ${{ env.IMAGE_PREFIX }}:${{ env.IMAGE_TAG }}
             ${{ env.IMAGE_PREFIX }}:latest
+
+      - name: Build binaries
+        run: make build-vct
+
+      - name: Upload binaries and their SHA to Github Release
+        uses: alexellis/upload-assets@0.3.0
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
@troyronda token needs to be set

Reply:
@soluchok https://docs.github.com/en/actions/reference/authentication-in-a-workflow